### PR TITLE
Add html pages to the project

### PR DESCRIPTION
**Pull request by Algor-Fernand**
Create the html pages for the home, about and services pages.

#Bundle2  Exercise 1

### CODE REVIEW

**Review by kaitcham — APPROVED** [review]
Hi, @Algor-Fernand 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit:

**Review by Yvad60 — CHANGES_REQUESTED** [review]
Hi @Algor-Fernand. Good job on the changes, I took a peek at your changes and the bits I saw looked good. Instead of putting your terminal logs in the PR description I would recommend creating a `README.md`  and paste your terminal logs there. For this check the video about how to submit exercises in the video playlist shared with you

With the current implementation after merging the current PR, you would loose all your changes because they would be closed with the PR itself

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
-This project will be used for Git exercise & practice.
\ No newline at end of file
+This project will be used for Git exercise & practice.
+
+#TerminalLogs BUNDLE2 Exercise 1
+
+´´´´bash
+DOLPHIX ELECTRONICS@DESKTOP-TJ49EDU MINGW64 ~/git-exercise (dev)
+$ git checkout ft/bundle-2
+Switched to branch 'ft/bundle-2'
```
Comment (Yvad60) [inline readme.md:10]: If you want to add a code block you use 3 backticks like ``` you can find backticks on the key above the tab key

**Review by Yvad60 — APPROVED** [review]
I left a comment about how to use code blocks properly but everything else looks good to me. Good job

**Pull request merged**